google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.storage;
+
+/**
+ * Enums for the storage classes.
+ * See https://cloud.google.com/storage/docs/storage-classes for details.
+ */
+public enum StorageClass {
```

Comment:
I don't know if we want to use an enum for this. We are using valueOf to convert from string to enum, which will throw an exception on an unexpected value. So if storage adds a new storage class, then we will be broken.